libs/index/odc/index/stac.py

```diff
@@ -86,14 +86,24 @@ def _get_stac_bands(
 ) -> Tuple[Document, Document]:
     bands = {}
     grids = {}
+    accessories = {}
+
     assets = item.get("assets", {})
 
+    def _get_path(asset):
+        path = asset["href"]
+        if relative:
+            path = Path(path).name
```

Comment:
We haven't seen any datasets with subfolders, have we?

Reply:
Not in the tests...